map_model/src/edits/mod.rs

```diff
@@ -771,18 +851,22 @@ impl Map {
             self.zones = Zone::make_all(self);
         }
 
-        (
-            effects.changed_roads,
-            effects.deleted_lanes,
-            effects.deleted_turns,
-            // Some of these might've been added, then later deleted.
-            effects
-                .added_turns
-                .into_iter()
-                .filter(|t| self.turns.contains_key(t))
-                .collect(),
-            effects.changed_intersections,
-        )
+        // Some of these might've been added, then later deleted.
+        retain_btreeset(&mut effects.added_turns, |t| self.turns.contains_key(t));
+
+        let mut more_changed_intersections = Vec::new();
```

Comment:
`apply_map_edits` in the UI did this previously. I realized we should just return the `EditEffects` struct instead of an ever-growing tuple, so moved that logic here.